server/src/main/java/io/littlehorse/common/LHConstants.java

```diff
@@ -52,6 +55,11 @@ public class LHConstants {
     public static final String DEFAULT_TENANT = "default";
     public static final String ANONYMOUS_PRINCIPAL = "anonymous";
 
+    public static final ServerACL ADMIN_ACL = ServerACL.newBuilder()
```

Comment:
It looks like ADMIN_ACL it is not a constant, ACLs will change depending on the Principals and they should be rehused for the different cases